codeworld-server/codeworld-server.cabal

```diff
@@ -24,20 +24,30 @@ Executable codeworld-server
     cryptonite,
     data-default,
     directory,
+    engine-io,
+    engine-io-snap,
     filepath,
     filesystem-trees,
+    funblocks-server,
```

Comment:
The file structure for funblocks and codeworld are different. So, the handlers for handling files in both have different logic. I thought it would be best to have a separate library for the same. But I am open up for suggestions